controllers/RecipesController.php

```diff
@@ -62,7 +62,7 @@ public function Overview(\Slim\Http\Request $request, \Slim\Http\Response $respo
 		return $this->AppContainer->view->render($response, 'recipes', [
 			'recipes' => $recipes,
 			'recipesResolved' => $recipesResolved,
-			'recipePositionsResolved' => $this->Database->recipes_pos_resolved(),
+			'recipePositionsResolved' => $this->Database->recipes_pos_resolved()->where('recipe_type', RecipesService::RECIPE_TYPE_NORMAL),
```

Comment:
Needs to be done based on the query parameter `include-internal` like above in line 25 - is only for dev/debugging purposes, but when supplied, also internal recipes (so all types) will be shown on the recipes page...